src/webextension/locales/en-US/manage.ftl

```diff
@@ -1,22 +1,22 @@
 toolbar-item-count = { $count ->
-   [one]   1 item
-  *[other] { $count } items
+   [one]   1 Entry
+  *[other] { $count } Entries
 }
 
-toolbar-add-item = Add Item
+toolbar-add-item = New Entry
 toolbar-go-home = Home
 toolbar-send-feedback = Feedback
 
-item-filter.placeholder = Filter…
+item-filter.placeholder = Search for an Entry
 
-item-summary-new-item = (new item)
-item-summary-no-title = (no title)
-item-summary-no-username = (no username)
+item-summary-new-item = New Entry
+item-summary-no-title = (No Entry Name)
+item-summary-no-username = (No Username)
 
 homepage-no-passwords =
   Welcome to Lockbox! I'm Lockie, and I'm here to help you lock
   up your passwords!
-  
+
```

Comment:
I'm pretty sure this will break things. The trailing whitespace is important for ensuring that fluent.js parses the entire body as one string.

Reply:
Atom was overly helpful.  Added back in next commit.